jekyll/_cci2/index.md

```diff
@@ -73,7 +73,7 @@ Even though there was no actual source code in your repo, and no actual tests co
 
 ## Breaking Your Build!
 
-Edit your `config.yml` file in the GitHub editor for simplicity and replace `echo "A first hello"` with `notacommand`. Click the **Commit change** button in the GitHub editor. When you navigate back to the {% comment %} TODO: Jobs {% endcomment %}Builds page in CircleCI, you will see that a new build was triggered. This build will fail with a red Failed button and will send you a notification email of the failure. 
+Edit your `config.yml` file in the GitHub editor for simplicity and replace `echo "A first hello"` with `notacommand`. Click the **Commit change** button in the GitHub editor. When you navigate back to the Jobs page in CircleCI, you will see that a new build was triggered. This build will fail with a red Failed button and will send you a notification email of the failure.
```

Comment:
Do we clear up somewhere that an individual run of a `job` is a `build`?

Reply:
No we do not! And my understanding is:

- a **build** has **jobs**.
- a **job** has **steps**.